src/modules/explorer/pages/ProposalDetails/index.tsx

```diff
@@ -187,6 +261,37 @@ export const ProposalDetails: React.FC = () => {
 
   const canVote = cycleInfo && proposal?.getStatus(cycleInfo.currentLevel).status === ProposalStatus.ACTIVE
 
+  const hasNFTs = useMemo(() => {
+    let NTFFound = false
+
+    transfers.map((transfer: Transfer) => {
+      if (
+        transfer.tokenId !== undefined &&
+        transfer.type === "FA2" &&
+        new BigNumber(transfer.tokenId).toNumber() !== 0
```

Comment:
Don't think this is a good way to check whether a token is NFT or not. THere's a tag for this probably in the API

Reply:
tokenId, type & amount are the only information I have available on the FE to identify the token type.
Also, this: 
https://tezos.stackexchange.com/questions/4259/how-to-separate-tokens-and-nftnonfungibletoken-in-result-of-tzkt-api

https://tezos.stackexchange.com/questions/3978/how-do-i-query-wallets-for-nfts